src/story.ts

```diff
@@ -390,6 +392,13 @@ export type ProjectAnnotations<
 > = BaseAnnotations<TRenderer, TArgs> & {
   argsEnhancers?: ArgsEnhancer<TRenderer, Args>[];
   argTypesEnhancers?: ArgTypesEnhancer<TRenderer, Args>[];
+
+  /**
+   * Function to be called once, before rendering any story. When the function is async, it will be awaited.
+   * A cleanup function may be returned. This function may only be defined globally.
```

Comment:
It might make sense to explain how/when beforeAll's cleanup function is called so integrators know what to expect

Reply:
Done.